compiler/compile/OMRCompilation.cpp

```diff
@@ -784,12 +782,23 @@ OMR::Compilation::getOSRInductionOffset(TR::Node *node)
    // If no induction after the OSR point, offset must be 0
    if (self()->getOSRTransitionTarget() != TR::postExecutionOSR)
       return 0;
+   
+   TR::Node *osrNode;
+   if (!self()->isPotentialOSRPoint(node, &osrNode))
+      {
+      TR_ASSERT(0, "getOSRInductionOffset should only be called on OSR points");
```

Comment:
What is the relationship between this point in the code and getOSRInductionOffset, i.e. I don't see a call immediately to that routine.